cla_backend/libs/eligibility_calculator/tests/cassettes/test_assessment_attribute_not_aggregated_no_income_low_capital_for_over_18_high_income.yml

```diff
@@ -6,7 +6,7 @@ interactions:
       {"gross": 20000.0, "receiving_only_statutory_sick_or_maternity_pay": false,
       "tax": -5.0, "benefits_in_kind": 0, "frequency": "monthly", "prisoner_levy":
       0, "national_insurance": -2.0, "student_debt_repayment": 0}}], "applicant":
-      {"date_of_birth": "1974-01-12"}, "assessment": {"submission_date": "2024-01-12",
+      {"date_of_birth": "1974-01-15"}, "assessment": {"submission_date": "2024-01-15",
```

Comment:
Could I please query this, how frequently are these values expected to be changed within these files? Are they re-generated every time the tests are ran?

Many thanks

Reply:
These files will only change when the tests are re-recorded (which will be very infrequently). This is caused by a combination of the default request matcher not including the body, and the tests not controlling the submission date (it defaults to today always)

Reply:
A couple of thoughts:
* The 'request matcher' should depend on the body
* Let's document (in readme):
  * Roughly how it works - when tests run, the requests to CFE-Civil are intercepted by VCR, and it returns the cached response, that is stored in this directory
  * When cla_backend changes, and the requests to CFE change, VCR will throw an error, so you need to re-record
  * Although CFE changes should always be backwards compatible, we should re-record every now and again, to test against the latest CFE responses. Command for doing that.
  * Docs for VCR are here: https://github.com/kevin1024/vcrpy/blob/v3.0.0/docs/configuration.rst (We use older version of vcr due to needing python2 support)

Reply:
This follow on work is now in a separate ticket: https://dsdmoj.atlassian.net/browse/LEP-484

Happy for this to be merged